app/_src/introduction/kuma-requirements.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Kuma Requirements
+---
+
+This page expose the different requirements to run {{site.mesh_product_name}}.
+
+## Architecture
+
+{{site.mesh_product_name}} supports machines with `x86_64` architecture and since `1.7.0` it's possible to run {{site.mesh_product_name}} on `arm64`.
```

Comment:
Unfortunately Kong Mesh 1.8.0 is based on Kuma 1.7.0 so you need an if statement here :(. Now the versions are equal so this won't be a problem in the future.

Reply:
Done

Reply:
I don't think it's correct. Here is the mapping from kuma to kong-mesh versions - https://github.com/Kong/kong-mesh/blob/cd0a8cb99a79a08759ccbd22279386eb8cb72804/.github/workflows/repository_dispatch.yaml#L25-L28 . So Kong Mesh supports arm64 from `1.8.0` and Kuma supports it from `1.7.0`. Maybe to not have this `if` block we should just stick with `1.8.0`.